client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-action-bar.tsx

```diff
@@ -0,0 +1,77 @@
+import { Icon, reusableBlock, chevronUp, chevronDown, closeSmall } from '@wordpress/icons';
+import classNames from 'classnames';
+
+type PatternActionBarProps = {
+	onReplace: () => void;
+	onDelete: () => void;
+	onOrderUp?: () => void;
+	onOrderDown?: () => void;
+	disableOrderUp?: boolean;
+	disableOrderDown?: boolean;
+	enableOrdering?: boolean;
+};
+
+const PatternActionBar = ( {
+	onReplace,
+	onDelete,
+	onOrderUp,
+	onOrderDown,
+	disableOrderUp,
+	disableOrderDown,
+	enableOrdering,
+}: PatternActionBarProps ) => {
+	return (
+		<div className="pattern-action-bar" role="menubar" aria-label="Pattern Actions">
+			{ enableOrdering && (
+				<div className="pattern-action-bar__block">
+					<div
```

Comment:
I think it has to be a button and we could just use the button from `@wordpress/components`

```js
import { Button } from `@wordpress/components;

<Button icon={ chevronUp } label={ label } iconSize={ size } />;
<Button icon={ chevronDown } label={ label } iconSize={ size } />;
<Button icon={ reusableBlock } label={ label } iconSize={ size } />;
<Button icon={ closeSmall } label={ label } iconSize={ size } />;
```